core/templates/dev/head/pages/exploration_player/ProgressNavDirective.js

```diff
@@ -100,11 +104,15 @@ oppia.directive('progressNav', [
           };
 
           $scope.shouldContinueButtonBeShown = function() {
+            if ($scope.activeCard.stateName === null) {
```

Comment:
Again this should be activeCard.isConceptCard()

Reply:
Changed it to isStateShowingConceptCard().